app/models/framework/definition/language.rb

```diff
@@ -7,10 +7,16 @@ def [](framework_short_name)
 
           fdl_filename = Rails.root.join("app/models/framework/definition/#{sanitized_short_name}.fdl")
 
-          raise ArgumentError, "#{framework_short_name}.fdl does not exist in models/framework/definition" \
-            unless File.exist?(fdl_filename)
+          fdl_source = if File.exist?(fdl_filename)
+                         File.read(fdl_filename)
+                       else
+                         Framework.find_by(short_name: framework_short_name)&.definition_source
```

Comment:
Is this always going to be stored as a full file path?

Reply:
@lozette Not sure I understand..? The default is to read from the filesystem until the point we move everything into the database.

Reply:
Doh I misunderstood, sorry